### ISSUE TITLE
incomplete video tutorial and errors

### ISSUE THREAD

**Issue by cerulliber** [Mozer/talk-llama-fast#35]
greetings,

1.I don't understand how to set environment variables. this isn't in written description, only in youtube video. exact time, up to 9:33:
https://youtu.be/0MEZ84uH4-E?t=542
unfortunately, your windows is in russian and it's 11 no 10 so may be differences
do I need to go to system properties>Advanced>Environment variables>new> add path to C:\talk-llama-fast-0.2.0\xtts\SillyTavern-Extras\tts_out ?

2.you also skipped  h264 codec. currently availabe, don't know which one to get:
http://ciscobinary.openh264.org/openh264-2.4.1-win32.dll.bz2
http://ciscobinary.openh264.org/openh264-2.4.1-win32.dll.signed.md5.txt
http://ciscobinary.openh264.org/openh264-2.4.1-win64.dll.bz2
http://ciscobinary.openh264.org/openh264-2.4.1-win64.dll.signed.md5.txt

3. where do I put models ggml-medium.en-q5_0 and mistral-7b-instruct-v0.2.Q5_0 ? straight into \talk-llama-fast-0.2.0 folder?
4. when running xtts_wav2lip.ba got this error C:\Users\xxx\miniconda3\envs\xtts\Lib\site-packages\pydantic\_internal\_fields.py:161: UserWarning: Field "model_name" has conflict with protected namespace "model_".

I can't thank you enough for your work!

**Comment by cerulliber** [comment]
WARNING: model is not multilingualrun: processing, 4 threads, lang = en, task = transcribe, timestamps = 0 ...

**Comment by cerulliber** [comment]
pls help. thanks!

**Comment by Wuzzooy** [comment]
1) if you read the tutorial it points to this link with a tutorial on how to install ffmpeg and set env variable
https://phoenixnap.com/kb/ffmpeg-windows.
It's just a path that will point to where you installed ffmpeg so it depends of where you install it so don't copy paste other people path.
for example you install direct in C:\ the path will look like this C:\ffmpeg\bin like show in the link but you are not forced to install it there if you don't want to.

2)If your windows is 64 bit which i guess it it, use win64.

3)put the model where you start script .bat

4)i've seen this warning which is a warning not an error but it will still work without issue so you can ignore it

I know that chatgpt can hallucinate but it can greatly help with stuff like this, yes AI can do more than "RP"

**Comment by cerulliber** [comment]
C:\t\xtts>conda create -n xtts
'conda' is not recognized as an internal or external command,
operable program or batch file.

tried update mini conda, tried install full conda, not worked

any help will be highly appreciated. I really want this installed. thanks!

**Comment by Wuzzooy** [comment]
Yeah this error is self explanatory, to build the tts wheel(coqui xtts) you need visual studio tools and yeah the link provided is the right one.
You can check visual studio installation by typing  cl in the terminal window
You should see something like this after pressing enter if it's correctly installed
```
Microsoft (R) C/C++ Optimizing Compiler Version 19.40.33812 for x64
Copyright (C) Microsoft Corporation.  All rights reserved.

usage: cl [ option... ] filename... [ /link linkoption... ]
```
If you still have issue with visual studio, follow this tutorial, since the tutorial is a bit old, the version of your visual studio will be different but it doesn't matter.
https://github.com/bycloudai/InstallVSBuildToolsWindows

git and visual studio are common tools that people use and have in this field so it's not easy to think and include every requirements that might be required in a tutorial when you already have them and you don't come across the error yourself.

**Comment by cerulliber** [comment]
thanks anyone, now video is really totally different!!!
